main.libsonnet

```diff
@@ -0,0 +1,25 @@
+{
+  File(name, content=""):: {
+    local file = self,
```

Comment:
I found the reference to `file` confusing on line 13, I think `self` would have been more clear. It made me think "what file?... oh, this file"

Reply:
Does this mean a `self` on line 13 would be referring to the `encodingStrategies`?